src/components/auth/login/Login.js

```diff
@@ -8,21 +14,50 @@ import { login } from "../../../store/actions";
 //import "../Auth.css"; //! commented out by Noor : "not used in the component"
 import "./Login.css"
 
+const config = {
+  apiKey: "AIzaSyDQLv5I4OQ8i0TxIaHRTkH40UEG3xef7oc",
```

Comment:
This needs to be in an env file

Reply:
Hey matt can you hide this comment or delete it? thanks